react-hyf/react-hyf-client/src/components/Week1.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+const ListItem = props => {
+  const { description, deadline } = props.listInfo;
+  return (
+    <div>
+      <li>
```

Comment:
Here it says an `li` inside of an `li`, but following the stack it's inside of a `div` then and `li`

<img width="554" alt="Screen Shot 2019-07-17 at 06 28 30" src="https://user-images.githubusercontent.com/25272822/61347151-20aa4c80-a85c-11e9-9fdc-5910f2e515ea.png">